src/main/java/org/mifos/connector/mpesa/flowcomponents/transaction/MpesaWorker.java

```diff
@@ -0,0 +1,89 @@
+package org.mifos.connector.mpesa.flowcomponents.transaction;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.camunda.zeebe.client.ZeebeClient;
+import org.apache.camel.CamelContext;
+import org.apache.camel.Exchange;
+import org.apache.camel.ProducerTemplate;
+import org.apache.camel.support.DefaultExchange;
+import org.mifos.connector.common.channel.dto.TransactionChannelRequestDTO;
+import org.mifos.connector.mpesa.dto.BuyGoodsPaymentRequestDTO;
+import org.mifos.connector.mpesa.utility.EntityMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+import java.util.Map;
+
+import static org.mifos.connector.mpesa.camel.CamelProperties.*;
+import static org.mifos.connector.mpesa.camel.CamelProperties.TRANSACTION_TYPE;
+import static org.mifos.connector.mpesa.camel.config.CamelProperties.BUY_GOODS_REQUEST_BODY;
+
+@Component
+public class MpesaWorker {
+
+    private final Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    @Autowired
+    EntityMapper<TransactionChannelRequestDTO, BuyGoodsPaymentRequestDTO>
+            channelRequestDTOToBuyGoodsPaymentRequestDTOEntityMapper;
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @Autowired
+    private ZeebeClient zeebeClient;
+
+    @Autowired
+    private ProducerTemplate producerTemplate;
+
+    @Autowired
+    private CamelContext camelContext;
+
+    @Value("${zeebe.client.evenly-allocated-max-jobs}")
+    private int workerMaxJobs;
+
+    @PostConstruct
+    public void setupWorkers() {
+
+        zeebeClient.newWorker()
+                .jobType("init-transfer")
```

Comment:
Are you sure the jobType is in line with the BPMN?

Reply:
yes, I have checked here --> https://github.com/openMF/ph-ee-env-template/blob/c064a25f5febab7348005db72c4e402a85283d90/orchestration/feel/mpesa-flow.bpmn#L17